### PR TITLE
[JW8-10037] Fix linting warnings

### PR DESCRIPTION
**Pull request by mamaddox**
### This PR will...

- Fix the ESLint warnings that appear when building

### Why is this Pull Request needed?

- It will make it easier to read the build logs

#### Addresses Issue(s):

[JW8-10037](https://jwplayer.atlassian.net/browse/JW8-10037)

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

**Comment by zetagame** [comment]
One missing:

`/var/lib/jenkins-slave/workspace/jw7-opensource/src/js/polyfills/webvtt.js
  724:79  warning  Infix operators must be spaced  space-infix-ops`

File: src/js/view/controls/controls.js [hunk 1]
```diff
@@ -110,7 +110,11 @@ export default class Controls extends Events {
                 this.trigger(DISPLAY_CLICK);
                 this.userActive(1000);
                 api.playToggle(reasonInteraction());
-                model.get('isFloating') ? this.wrapperElement.focus() : this.playerContainer.focus();
+                if (model.get('isFloating')) {
+                    this.wrapperElement.focus();
+                } else {
+                    this.playerContainer.focus();
+                }
```
Comment (robwalch) [inline src/js/view/controls/controls.js]: We're doing the same thing on line 214 with:
```
const focusElement = model.get('isFloating') ? this.wrapperElement : this.playerContainer;
focusElement.focus();
```

Comment (mamaddox) [reply src/js/view/controls/controls.js]: good call! I'll keep a look out for things like this.

File: src/js/view/controls/controls.js [hunk 2]
```diff
@@ -101,6 +101,7 @@ export default class Controls extends Events {
         this.logo = this.playerContainer.querySelector('.jw-logo');
 
         const touchMode = model.get('touchMode');
+        const floatingElement = model.get('isFloating') ? this.wrapperElement : this.playerContainer;
```
Comment (mamaddox) [inline src/js/view/controls/controls.js]: Not sure about the naming for this const?

Comment (robwalch) [reply src/js/view/controls/controls.js]: lgtm!

**Pull request merged**